spec/formatting.md

```diff
@@ -16,7 +16,7 @@ The formatting of a _message_ is defined by the following operations:
   For a message with no _selectors_, this is simple as there is only one _pattern_.
   With _selectors_, this will depend on their resolution.
 
-- **_<dfn>Formatting</dfn>_** takes the resolved values of the selected _pattern_,
+- **_<dfn>Formatting</dfn>_** takes the _resolved values_ of the selected _pattern_,
```

Comment:
What is the "resolved values" of a pattern? Do you mean a the resolved values **of the placeholders** in a selected pattern?

That is false, however. It takes not the "resolved values", but rather the *formatted values* of the placeholders.